api/tests/acceptance/application/session/session-controller-patch-unpublish-session_test.js

```diff
@@ -0,0 +1,116 @@
+const {
+  expect, generateValidRequestAuthorizationHeader, databaseBuilder, knex,
+} = require('../../../test-helper');
+const createServer = require('../../../../server');
+
+describe('PATCH /api/admin/sessions/:id/unpublish', () => {
+  let server;
+  const options = { method: 'PATCH' };
+  let userId;
+
+  beforeEach(async () => {
+    server = await createServer();
+  });
+
+  context('when user does not have the role PIX MASTER', () => {
+
+    beforeEach(() => {
+      userId = databaseBuilder.factory.buildUser().id;
+      return databaseBuilder.commit();
+    });
+
+    it('should return a 403 error code', async () => {
+      // given
+      options.url = '/api/admin/sessions/1/unpublish';
+      options.headers = { authorization: generateValidRequestAuthorizationHeader(userId) };
+
+      // when
+      const response = await server.inject(options);
+
+      // then
+      expect(response.statusCode).to.equal(403);
+    });
+
+  });
+
+  context('when user has role PixMaster', () => {
+
+    beforeEach(() => {
+      // given
+      userId = databaseBuilder.factory.buildUser.withPixRolePixMaster().id;
+      options.headers = { authorization: generateValidRequestAuthorizationHeader(userId) };
+      return databaseBuilder.commit();
+    });
+
+    context('when the session id has an invalid format', () => {
+
+      it('should return a 400 error code', async () => {
+        // given
+        options.url = '/api/admin/sessions/any/unpublish';
+
+        // when
+        const response = await server.inject(options);
+
+        // then
+        expect(response.statusCode).to.equal(400);
+      });
+    });
+
+    context('when the session id is a number', () => {
+
+      context('when the session does not exist', () => {
+
+        it('should return a 404 error code', async () => {
+          // given
+          options.url = '/api/admin/sessions/1/unpublish';
+
+          // when
+          const response = await server.inject(options);
+
+          // then
+          expect(response.statusCode).to.equal(404);
+        });
+      });
+
+      context('when the session exists', () => {
```

Comment:
```suggestion
      context('when the session exist', () => {
```

Reply:
"exists" à la troisième personne prend bien un s en anglais : 
https://www.theconjugator.com/la/conjugaison/du/verbe/to+exist.html